Sources/Preferences/util.swift

```diff
@@ -1,10 +1,36 @@
 import Cocoa
 
-struct System {
-	/// Get a system localized string
-	/// Use https://itunes.apple.com/no/app/system-strings/id570467776 to find strings
-	static func localizedString(forKey key: String) -> String {
-		return Bundle(for: NSApplication.self).localizedString(forKey: key, value: nil, table: nil)
+class System {
+	private static let preferencesLocalized = [ "ar": "تفضيلات", "ca": "Preferències", "cs": "Předvolby", "da": "Indstillinger", "de": "Einstellungen", "el": "Προτιμήσεις", "en": "Preferences", "en_AU": "Preferences", "en_GB": "Preferences", "es": "Preferencias", "es_419": "Preferencias", "fi": "Asetukset", "fr": "Préférences", "fr_CA": "Préférences", "he": "העדפות", "hi": "प्राथमिकता", "hr": "Postavke", "hu": "Beállítások", "id": "Preferensi", "it": "Preferenze", "ja": "環境設定", "ko": "환경설정", "ms": "Keutamaan", "nl": "Voorkeuren", "no": "Valg", "pl": "Preferencje", "pt": "Preferências", "pt_PT": "Preferências", "ro": "Preferințe", "ru": "Настройки", "sk": "Nastavenia", "sv": "Inställningar", "th": "การตั้งค่า", "tr": "Tercihler", "uk": "Параметри", "vi": "Tùy chọn", "zh_CN": "偏好设置", "zh_HK": "偏好設定", "zh_TW": "偏好設定" ]
+	private static let preferencesEllipsizedLocalized = ["ar": "تفضيلات…", "ca": "Preferències…", "cs": "Předvolby…", "da": "Indstillinger…", "de": "Einstellungen…", "el": "Προτιμήσεις…", "en": "Preferences…", "en_AU": "Preferences…", "en_GB": "Preferences…", "es": "Preferencias…", "es_419": "Preferencias…", "fi": "Asetukset…", "fr": "Préférences…", "fr_CA": "Préférences…", "he": "העדפות…", "hi": "प्राथमिकता…", "hr": "Postavke…", "hu": "Beállítások…", "id": "Preferensi…", "it": "Preferenze…", "ja": "環境設定…", "ko": "환경설정...", "ms": "Keutamaan…", "nl": "Voorkeuren…", "no": "Valg…", "pl": "Preferencje…", "pt": "Preferências…", "pt_PT": "Preferências…", "ro": "Preferințe…", "ru": "Настройки…", "sk": "Nastavenia…", "sv": "Inställningar…", "th": "การตั้งค่า…", "tr": "Tercihler…", "uk": "Параметри…", "vi": "Tùy chọn…", "zh_CN": "偏好设置…", "zh_HK": "偏好設定⋯", "zh_TW": "偏好設定⋯"]
```

Comment:
Can you put each entry on separate lines?

Reply:
done